pom.xml

```diff
@@ -101,6 +112,25 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.8</version>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <version>${findbugs-maven-plugin.version}</version>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
+          <failOnError>${findbugs-maven-plugin.failOnError}</failOnError>
```

Comment:
Do not hijack the existing user property. If a parent has forced this to `true` or `false` and you want to switch it back, then use `${findbugs.failOnError}` not a random property name that does not fit in with the existing maven plugin documentation